stable/stolon/templates/create-cluster-job.yaml

```diff
@@ -0,0 +1,50 @@
+{{ if .Release.IsInstall }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "stolon.fullname" . }}-create-cluster
+  labels:
+    app: {{ template "stolon.name" . }}
+    chart: {{ template "stolon.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{ template "stolon.fullname" . }}
+        release: {{ .Release.Name }}
+      annotations:
+        {{- if eq .Values.store.backend "etcd" }}
+        pod.beta.kubernetes.io/init-containers: '[
```

Comment:
Don't use annotations for init container any more.

Reply:
fixed